src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/model/community/GroupedDocuments.kt

```diff
@@ -0,0 +1,40 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.model.community
+
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class GroupedDocuments(
+  val documents: List<OffenderLevelDocument>,
+  val convictions: List<ConvictionDocuments>
+)
+
+data class OffenderLevelDocument(
+  val id: UUID,
+  val documentName: String,
+  val author: String,
+  val type: DocumentType,
+  val extendedDescription: String,
```

Comment:
I think `extendedDescription` needs to be optional as it is often missing...

Reply:
I think we also need an optional `parentPrimaryKeyId` on the `OffenderLevelDocument` as this can be present here as well as within the conviction docs

Reply:
sorted 👍